.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
```

Comment:
I'm not sure who uses .editorconfig file, but why use Unix-style line endings and 2 char indent by default? Seems to mek the default should be CRLF and 4 space indent.

Reply:
Will update the LF to CRLF.
Both Visual Studio and Visual Studio Code use EditorConfig.

Keeping the indent as size 2 for MSBuild files is convenient for readability.
For sure, CPP files in this code base use size 4.

Reply:
How about making the default indent 4 spaces, and specifying the 2 space indent for the MSBuild file extensions specifically? There are other files besides .h and .cpp for C++ source files and it will be good to use 4 spaces consistently.

Reply:
Alright.

Reply:
Done.